src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,10 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
+        logger.info("Login method was called. Params ={}", login);
```

Comment:
Let's make our log messages more informative by including method parameters in the log output. For example: logger.info("Method login was called with login={}", login);